package.json

```diff
@@ -8,7 +8,7 @@
     "npm": "^6.4.1"
   },
   "scripts": {
-    "copy-views": "cp -R server/views/ dist/server/views/",
+    "copy-views": "cp -R server/views/ dist/server/views/ && cp -R mock-server/routes/images dist/mock-server/routes",
```

Comment:
meh.. this seems a bit messy

Reply:
I know.  
mock-server needs to serve this image. Without it you can't print a licence against the mocks.
Its there so that you can run mock-server from a licences docker image using docker-compose.
It is one small image.
I could add a separate script and add that to the sequence called from 'build'.  Or have a separate Docker file for stubs. (More complicated but separate).  Any suggestions?

Reply:
nah that's alright...